provider-gui/app/provider-profile/profile-environment/page.tsx

```diff
@@ -0,0 +1,14 @@
+'use client'
+
+import { environmentMock } from '@/mocks/profile-mocks'
```

Comment:
can you rename these to be less verbose? i.e. provider-profile/profile-env... is repeating the word profile. Could do this in the component names too